CHANGELOG.md

```diff
@@ -1,10 +1,47 @@
-# Change Log: Trust-DNS
+# Change Log: Hickory DNS (f.k.a. Trust-DNS)
```

Comment:
I'd just expand the `f.k.a.`, might not be a well-known acronym for everyone.

Reply:
replaced with "formerly"